client/src/contexts/DownloadOptionsContext.js

```diff
@@ -34,7 +34,8 @@ export const DownloadOptionsContextProvider = ({
   useEffect(() => {
     if (resource) {
       setComputedFiles(() => {
-        if (!resourceAttribute) return resource.computed_files
+        if (!resourceAttribute)
+          return resource.computed_files.filter((f) => f.modality) // Exclude the metadata_only file
```

Comment:
```suggestion
          return resource.computed_files.filter((f) => !f.metadata_only)
```

One last change to make here, since we want to omit computed files where `metadata_only` is `true` we should just do this instead of checking the truthiness of `modality`.

Reply:
You can put a comment here saying that modality and format would be empty when this is true if you find that helpful, but I think this change makes the implication more obvious.

Reply:
I've also added the comment you suggested, see the commit at 683d3378f21087f42fb6efe96b8dd418573b326d.